### PR TITLE
set up model-data comparison

### PR DESCRIPTION
**Pull request by Zhanzhang-2020**

### CODE REVIEW

File: src/DE.jl [hunk 1]
```diff
@@ -199,6 +199,14 @@ function save_params(optt)
     writedlm("epss.csv", ε, ',')
 end
 
+" Set up model-data comparison "
+function comparison(optt)
+    pIn = Optim.minimizer(optt)
```
Comment (aarmey) [inline src/DE.jl:204]: Can this just take the parameters rather than running the optimization? This will make it easier to look after fitting.

Comment (Zhanzhang-2020) [reply src/DE.jl:204]: I see, so we may consider return the parameters instead of optt in function 'runOptim'?

Comment (Zhanzhang-2020) [reply src/DE.jl:204]: Then it's easier to store and access in following function?

Comment (aarmey) [reply src/DE.jl:204]: Think that makes sense.

**Pull request merged**